src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -93,21 +93,31 @@ class LogLine():
     def __init__(self, line):
         fields = line.split()
         # Work out the end of the fixed-width portion, and the beginning of the
-        # message.  The hostname and pid fields are both variable width
-        idx = 29 + len(fields[1]) + len(fields[2])
-        pidtid = fields[2][5:-1]
+        # message. The hostname, pid, fac and level fields are all variable width.
+        idx = 0
+        for i in range(4):
+            idx += len(fields[i]) + 1
+        # assuming (mst.oflags & DLOG_FLV_FAC) always true in src/gurt/dlog.c: 664
```

Comment:
I don't think this should reference line numbers because those will change at some point and nobody will know to update this

Reply:
Done